src/worker/ingestion/process-event.ts

```diff
@@ -288,6 +289,11 @@ export class EventsProcessor {
         const oldPerson = await this.db.fetchPerson(teamId, previousDistinctId)
         const newPerson = await this.db.fetchPerson(teamId, distinctId)
 
+        if (oldPerson?.is_identified) {
```

Comment:
I think this case might be too early in this chain but it might be worth it to chat through the consequences of this.

Reply:
test: timing/concurrency issues